src/utils/stringifyVariables.ts

```diff
@@ -3,6 +3,8 @@ const seen = new Set();
 const stringify = (x: any): string => {
   if (x === undefined) {
     return '';
+  } else if (x && x.toJSON) {
```

Comment:
can we just move this after `x !== 'object` for then we can probably remove `x &&`

Reply:
I moved it on the last position, otherwise it would be called when `x === null`